modules/dashboard/help/dashboard.md

```diff
@@ -2,39 +2,32 @@
-
-Key user information is displayed at the right side of the Menu Bar, including the user's name and the sites to which the user belongs.
-
-To log out from LORIS, click on the user's name and select "Log Out" from the dropdown menu, or select "User Preferences" to update user information, profile settings, and to change your password.
-
-Click on any of the six main menus to access LORIS modules.
+Need to update your account information, such as your password or other profile settings? Click on your name in the top right corner and click **User Preferences**. 
 
 ## Dashboard Features
 
-The dashboard is composed of panels showing at-a-glance status of the LORIS study. It has panels for each of: Recruitment, Study Progression, My Tasks, Document Repository Notifications, and Behavioral Feedback Notifications. Each panel can be hidden by clicking the arrow located in the top right corner.
+This is where you’ll find an at-a-glance status of your LORIS study. Note that all data shown in these sections are from ineligible, excluded, and consent-withdrawn candidates. You can temporarily hide any of the sections from view by clicking the arrow located in the top right corner. We’ve broken down the different sections below:
 
-### Recruitment
+#### Recruitment
 
-View how many candidates have been recruited for a given study.  Click the "Views" button in the right-hand corner for a breakdown by site. Note that charts include data from ineligible, excluded, and consent-withdrawn candidates.
+Number of candidates that have been recruited for a given study. Click **Views** to narrow this view by site or project.
 
-### Study Progression:
+#### Study Progression
 
-View a timeline of scans collected by each study site, or a timeline of candidate recruitment per site. Zoom in on the chart to view detailed data points. Note that charts include data from ineligible, excluded, and consent withdrawn candidates.
+A timeline of scans collected by each study site, or a timeline of candidate recruitment per site. Zoom in on the chart to view detailed data points.
 
-### My Tasks
+#### My Tasks
 
 User-specific list of outstanding tasks such as data entry conflicts, incomplete forms, or new and pending scans awaiting review. Click on any task panel to jump to the appropriate module.
 
-### Document Repository Notifications
+#### Document Repository Notifications
 
-View recently uploaded documents for your study or site user's study or site.  Click on a document name to download the file directly from the Dashboard. Click on "Document Repository" to view the complete list of available documents.
+Recently-uploaded documents for your study or site user's study or site. Click on a document name to download the file directly from the Dashboard. Click **Document Repository** to view the complete list of available documents.
```

Comment:
> for your study or site user's study or site

This sentence is a little hard to parse imo. Otherwise looks good!

Reply:
thanks, done!